site/docs/latest/cookbooks/tiered-storage.md

```diff
@@ -17,42 +19,63 @@ A topic in Pulsar is backed by a log, known as a managed ledger. This log is com
 
-The configured S3 bucket must exist before attempting to offload. If it does not exist, the offload operation will fail.
+## Configuring for S3 and GCS in the broker
 
-Pulsar users multipart objects to update the segment data. It is possible that a broker could crash while uploading the data. We recommend you add a lifecycle rule your S3 bucket to expire incomplete multipart upload after a day or two to avoid getting charged for incomplete uploads.
+Offloading is configured in ```broker.conf```. 
 
-### Configuring the broker
+At a minimum, the administrator must configure the driver, the bucket and the authenticating.  There is also some other knobs to configure, like the bucket regions, the max block size in backed storage, etc.
 
-Offloading is configured in ```broker.conf```. 
+### Configure the driver
 
-At a minimum, the user must configure the driver, the region and the bucket.
+Currently we support driver of types: { "S3", "aws-s3", "google-cloud-storage" }, 
+{% include admonition.html type="warning" content="The chars are case ignored for driver's name. "s3" and "aws-s3" are similar, with "aws-s3" you just don't need to define the url of the endpoint because it will know to use `s3.amazonaws.com`." %}
 
 ```conf
 managedLedgerOffloadDriver=S3
-s3ManagedLedgerOffloadRegion=eu-west-3
+```
+
+### Configure the Bucket
+
+On the broker, the administrator must configure the bucket or credentials for the cloud storage service. The configured bucket must exist before attempting to offload. If it does not exist, the offload operation will fail.
+
+- Regarding driver type "S3" or "aws-s3", the administrator should configure `s3ManagedLedgerOffloadBucket`.
+
+```conf
 s3ManagedLedgerOffloadBucket=pulsar-topic-offload
 ```
 
-It is also possible to specify the s3 endpoint directly, using ```s3ManagedLedgerOffloadServiceEndpoint```. This is useful if you are using a non-AWS storage service which provides an S3 compatible API. 
+- While regarding driver type "google-cloud-storage", the administrator should configure `gcsManagedLedgerOffloadBucket`.
+```conf
+gcsManagedLedgerOffloadBucket=pulsar-topic-offload
+```
 
-{% include admonition.html type="warning" content="If the endpoint is specified directly, then the region must _not_ be set." %}
+### Configure the Bucket Region
 
-{% include admonition.html type="warning" content="The broker.conf of all brokers must have the same configuration for driver, region and bucket for offload to avoid data becoming unavailable as topics move from one broker to another." %}
+Bucket Region is the region where bucket located. 
 
-Pulsar also provides some knobs to configure the size of requests sent to S3.
+Regarding AWS S3, the default region is `US East (N. Virginia)`. Page [AWS Regions and Endpoints](https://docs.aws.amazon.com/general/latest/gr/rande.html) contains more information.
```

Comment:
The page is jumping back and forth between GCS and AWS a lot. For a user, this is very confusing. A user is either going to care about GCS or AWS, and not give a fig about the other. So, for the things you need to get up and running, they should be groups together.

In other words, the sections should be
- Configuring the Driver // s3 or gcs
- S3
   - configuration of region, bucket and credentials
   - note at end about setting the endpoint explicitly
- GCS
   - configuration of region, bucket and credentials
- Extra options
   - for the block size stuff, etc, that people are rarely going to touch

Reply:
Thanks, Agreed that most the user cares 1 of the 2 types. Current sections is following your original view, It seems be clear from the view of each setting:  in each setting, S3 first introduced, then GCS. I would like to keep the current view.  

Most of the settings is similar(except the endpoint).  And most words of each sections is about explanation the meaning of settings,  less words for the setting. If split S3 and GCS, there will be some dup of the explanation.